### PR TITLE
Sheena O'Connell keynote

### PR DESCRIPTION
**Pull request by jcjudkins**
Need to upload picture.

### CODE REVIEW

File: src/_content/posts/2021-07-23-sheena-oconnel.md
Comment (tim-schilling) [inline src/_content/posts/2021-07-23-sheena-oconnel.md:1]: @drewbrew @sodevious @jcjudkins This generates the URL news/sheena-oconnel/. So it doesn't seem like we need the date in the filename. But I do like having that organization in our filesystem. Was there a plan for this when building the site?

@jcjudkins ~~I just noticed we're missing an "L" in her name on the file.~~ And maybe we want to make it, "introducing-keynote-sheena-oconnell" so the URL is more descriptive? I'm not super tied to that idea, just curious of your thoughts.

**Review by tim-schilling — CHANGES_REQUESTED** [review]
~~Filename needs an extra L. Marking it as red so we don't miss it.~~

File: src/_content/posts/2021-07-23-sheena-oconnel.md
Comment (drewbrew) [reply src/_content/posts/2021-07-23-sheena-oconnel.md:1]: +1 to the more descriptive slug.

The naming question is one for @mtrythall, but I don't have any particular attachment to one method or another.

File: src/_content/posts/2021-07-23-sheena-oconnell.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+title: "Announcing Our Keynotes: Sheena O'Connell"
+---
+
+### We're honored to announce our first keynote speaker for DjangoCon US 2024: Sheena O'Connell!
+
+We had the opportunity to sit down with Sheena and ask her a few questions about her work, her upcoming talk, and what she's looking forward to at DjangoCon US 2024.
+
+## How Do You Use Django?
+
+>Currently, I use it at work. We have a learner management system built in Django. I spoke about it at DjangoCon 2022. That project is still running, but it's changed a bit. We have moved away from a React-based frontend to something built with HTMX, and it's way nicer.
```
Comment (drewbrew) [inline src/_content/posts/2021-07-23-sheena-oconnell.md]:
```suggestion
>Currently, I use it at work. We have a learner management system built in Django. I spoke about it at DjangoCon US 2022. That project is still running, but it's changed a bit. We have moved away from a React-based frontend to something built with HTMX, and it's way nicer.
```

File: src/_content/posts/2021-07-23-sheena-oconnell.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+## What Are Your Favorite Topics at Conferences?
+
+>I like being introduced to things I can use and don't yet know about. For example, I've found previous talks on HTMX and Playwright really useful. I use those tools now. I really like anything that is actionable. And I enjoy good stories and new ways of looking at things. If I can leave curious and inspired, then I'm happy.
+
+## Your Involvement with Conferences
+
+>I've presented at many places since my first conference, including DjangoCon Africa, DjangoCon Europe, and local conferences like PyCon Namibia and PyCon South Africa. Recently, my work has shifted more towards teaching teachers rather than directly working on hard tech. This involves dealing with stakeholders and non-developers, which requires a lot of learning and adaptation.
+
+## Building Communities and Future Plans
```
Comment (drewbrew) [inline src/_content/posts/2021-07-23-sheena-oconnell.md]: Phrase these two heaadings as questions for parallelism

**Review by drewbrew — COMMENTED** [review]
couple of minor tweaks

**Comment by drewbrew** [comment]
@tim-schilling can you fill in https://github.com/djangocon/2024.djangocon.us/blob/main/src/_content/schedule/talks/2024-09-23-09-15-00-t0-keynote-to-be-announced-monday.md with Sheena's details (and create presenter files for her) or do I need to do that?

File: src/_content/posts/2021-07-23-sheena-oconnel.md
Comment (jcjudkins) [reply src/_content/posts/2021-07-23-sheena-oconnel.md:1]: Ahhh good call. I'll updated the file name. Happy to put the date back in for organization if its ok.

**Comment by tim-schilling** [comment]
@drewbrew I can figure it out

**Comment by tim-schilling** [comment]
@drewbrew @jcjudkins Does Sheena have a talk or any content to put into the keynote notes? Right now it's all empty.

File: src/_content/posts/introducing-keynote-sheena-oconnell.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
+---
+author: DjangoCon US Communications Team
+category: General
+date: 2024-07-23 06:00:00
+cover:
+    url: /assets/img/blog/sheena-oconnel.jpg
+    alt: "Sheena O'Connel"
```
Comment (tim-schilling) [inline src/_content/posts/introducing-keynote-sheena-oconnell.md]:
```suggestion
    alt: "Sheena O'Connell"
```

File: src/_content/posts/introducing-keynote-sheena-oconnell.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
+---
+author: DjangoCon US Communications Team
+category: General
+date: 2024-07-23 06:00:00
+cover:
+    url: /assets/img/blog/sheena-oconnel.jpg
```
Comment (tim-schilling) [inline src/_content/posts/introducing-keynote-sheena-oconnell.md]:
```suggestion
    url: /assets/img/blog/sheena-oconnell.jpg
```

Requires a filename change for the image.

**Review by tim-schilling — COMMENTED** [review]
A few more small things

**Review by tim-schilling — APPROVED** [review]
Confirmed all the pages are rendering as we expect them to. Thank you @jcjudkins!

**Comment by drewbrew** [comment]
> @drewbrew @jcjudkins Does Sheena have a talk or any content to put into the keynote notes? Right now it's all empty.

You can use the abstract she included in the original email I forwarded to you two if that works

File: src/_content/schedule/talks/2024-09-23-09-15-00-t0-keynote-sheena-oconnell.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
+---
+category: talks
+datetime: 2024-09-23 09:15:00-04:00
+difficulty: All
+end_datetime: 2024-09-23 10:00:00-04:00
+permalink: /talks/keynote-monday/
+presenter_slugs: 
+- sheena-oconnell
+room: Junior Ballroom
+title: Keynote - Sheena O'Connell (Monday)
```
Comment (drewbrew) [inline src/_content/schedule/talks/2024-09-23-09-15-00-t0-keynote-sheena-oconnell.md]:
```suggestion
title: Keynote - Power to the People who Teach the People
```

I don't know if her title ever made it to you, but here it is

**Review by drewbrew — APPROVED** [review]

File: src/_content/posts/introducing-keynote-sheena-oconnell.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
+---
+author: DjangoCon US Communications Team
+category: General
+date: 2024-07-23 06:00:00
```
Comment (tim-schilling) [inline src/_content/posts/introducing-keynote-sheena-oconnell.md]:
```suggestion
date: 2024-07-24 06:00:00
```

**Pull request merged**